spec/features/admin_user_create_events_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+require 'capybara/rspec'
+require_relative '../support/features/admin_user_helpers'
+
+feature 'Create events' do
```

Comment:
We must use System tests introduced in Rails 5.1
Your test should look like this after changes:
```ruby
require 'rails_helper'

RSpec.describe "Admin panel: Events creation", type: :system do
# ...
end
```